File: java/utility/ResultBody.java
package voxgig.bluefinshieldconexsdk.utility;

import voxgig.bluefinshieldconexsdk.core.Context;
import voxgig.bluefinshieldconexsdk.core.Response;
import voxgig.bluefinshieldconexsdk.core.Result;

final class ResultBody {

  private ResultBody() {}

  static Result resultBody(Context ctx) {
    Response response = ctx.response;
    Result result = ctx.result;

    if (result != null) {
      if (response != null && response.jsonFunc != null && response.body != null) {
        result.body = response.jsonFunc.get();
      }
    }

    return result;
  }
}

File: .sdk/tm/php/utility/ResultBody.php
<?php
declare(strict_types=1);

// BluefinShieldconex SDK utility: result_body

class BluefinShieldconexResultBody
{
    public static function call(BluefinShieldconexContext $ctx): ?BluefinShieldconexResult
    {
        $response = $ctx->response;
        $result = $ctx->result;
        if ($result && $response && $response->json_func && $response->body) {
            $result->body = ($response->json_func)();
        }
        return $result;
    }
}
